src/__tests__/courseDetails.test.js

```diff
@@ -0,0 +1,37 @@
+import { Provider } from 'react-redux';
+import userEvent from '@testing-library/user-event';
+import { render, cleanup, act } from '@testing-library/react';
+import React from 'react';
+import store from '../../src/store';
+import CourseDetails from '../components/pages/AdminHome/CourseDetails';
+
+afterEach(() => {
+  cleanup();
+});
+
+jest.mock('@okta/okta-react', () => ({
+  useOktaAuth: () => {
+    return {
+      authState: {
+        isAuthenticated: true,
+      },
+      authService: {
+        getUser: () => Promise.resolve({ name: 'sara' }),
+      },
+    };
+  },
+}));
+
+describe('Course Details is a connected React-Redux Component', () => {
+  let { component, getByText } = render(
+    <Provider store={store}>
+      <CourseDetails />
+    </Provider>
+  );
+  it('Manage Course Resources button works without errors', () => {
+    const button = getByText(/Manage Course Resources/i);
+    userEvent.click(button);
+
+    expect(button.textContent).toBe('Manage Course Resources');
```

Comment:
You are expecting the buttons text content to be the same as the text you got the button by, after the button has been clicked. This isn't really testing anything. In general, we only want to test things that change. If a user clicks the button, what should happen if no errors occur? That should be answered in the expected.

After checking the `CourseDetails.js` component inside `AdminHome`, nothing actually happens when the button is clicked. You should still expect something to happen, have the test fail, and implement the missing feature so that it passes. 

For example, maybe on click, the user gets pushed to a different page. What could you expect to find on that page?